src/models/deployers/SubscriptionsParamsManager.js

```diff
@@ -38,30 +38,30 @@ module.exports = class extends BaseDeployer {
     const agentCallsScript = []
 
     // fee token and amount
-    if (this.feeToken) {
-      this._addActionToCallsScript(agentCallsScript, aragonNetworkDAO.agent, this._setFeeToken(subscriptions.address, subscriptions.feeToken, subscriptions.feeAmount))
+    if (this.feeToken && this.feeAmount) {
+      this._addActionToCallsScript(agentCallsScript, aragonNetworkDAO.agent, this._setFeeToken(subscriptions.address, this.feeToken, this.feeAmount))
     } else if (this.feeAmount) {
-      this._addActionToCallsScript(agentCallsScript, aragonNetworkDAO.agent, this._oneParamSetter(subscriptions.address, 'setFeeAmount', subscriptions.feeAmount))
+      this._addActionToCallsScript(agentCallsScript, aragonNetworkDAO.agent, this._oneParamSetter(subscriptions.address, 'setFeeAmount', this.feeAmount))
```

Comment:
Oh, I see, so you are passing these config values as parameters in the CLI? Why doing it differently from all other commands in the repo? Isn't it better to have in the config file, so it gets saved in git history. I have been referring to `court.mainnet.js` a lot to point people to Court config values. Doing it as CLI params seems too ephemeral to me, and inconsistent with what we ahve doing so far.

Reply:
I thought that as well, but I was simply following what you implemented (see [here](https://github.com/aragon/aragon-network-deploy/pull/49/files#diff-6112a383f7a0c35c0d46686dd27140deR7-R16)), and it wasn't working as it was cause there was no params in the config file (see [here](data/config/susbcriptions-params-management.js)). If that was the intention, it wasn't clear honestly.

Reply:
> I was simply following what you implemented (see here)

This are just boolean flags to signal which setter you want to call.

> there were no params in the config file (see here)

That link is broken, but if you see [here](https://github.com/aragon/aragon-network-deploy/blob/master/data/config/subscriptions-params-management.js#L1), I was getting the config from the court file, to avoid duplication.

Reply:
Fixed the [link](https://github.com/aragon/aragon-network-deploy/pull/49/files#diff-85b0383d0ffb73a3939384fc01fc49d7) now.

Honestly, based on the given config file, it wasn't clear the intention was to change the original court config to do this. However, I think I would prefer having a config file only for this instead of tweaking the original one and signaling through flags which values should be changed, we can read a subscriptions config file and update whatever is in there directly. WDYT?

Reply:
Well, I really like having `court.mainnet.js` as a "single source of truth". So far I have been using [this link](https://github.com/aragon/aragon-network-deploy/blob/master/data/config/court.mainnet.js) myself when in doubt, and giving it to people who were asking about court parameters.
Looking at git history over that file we would be able to see all the changes that had happened along Court history.